tests/unit/helpers.js

```diff
@@ -1312,6 +1312,17 @@ export function fakeCookies(overrides = {}) {
   };
 }
 
+export function fakeCookiesWithJest(overrides = {}) {
```

Comment:
`fakeCookies` is used by other tests that have not been converted yet, so I created this version for the updated tests. Once everything that uses this has been converted we can delete `fakeCookies` and rename `fakeCookiesWithJest` to `fakeCookies`.

Reply:
I did consider it, and didn't do it for exactly the reason you state: trying to keep the current diff smaller. Ultimately it's probably the same amount of effort, but I figured a final PR to just rename everything would be easier to review.